api-reference/10 UI Widgets/dxTreeView/1 Configuration/createChildren.md

```diff
@@ -81,6 +81,69 @@ The following code shows how to use this function with a remote service:
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div>
+            <DxTreeView
+            :create-children="createChildren"
+            :root-value="''"
+            data-structure="plain"
+            />
+        </div>
+    </template>
+    <script>
+
+    import DxTreeView from 'devextreme-vue/tree-view';
+    import 'whatwg-fetch';
+
+    export default {
+        components: {
+            DxTreeView
+        },
+        methods: {
+            createChildren: function(parent) {
+            let parentId = parent ? parent.itemData.id : '';
+
+            return fetch(`https://js.devexpress.com/Demos/Mvc/api/TreeViewData?parentId=${parentId}`)
+                .then(response => response.json())
+                .catch(() => { throw 'Data Loading Error'; });
+            }
+        }
+    };
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import TreeView from 'devextreme-react/tree-view';
+    import 'whatwg-fetch';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <React.Fragment>
```

Comment:
The same note as about the div. Is React.fragment really required?

Reply:
Yep, you are absolutely right - it's redundant.
Fixed  - https://github.com/DevExpress/devextreme-documentation/pull/1555/commits/27e99f72319e69d41644da6d98ce34d858fe67ba